demo/addons/fmod/nodes/FmodSoundEffect.gd

```diff
@@ -0,0 +1,81 @@
+tool
```

Comment:
This is where the magic happens: using `tool` makes sure that Godot uses this script also within the editor and it will auto-modify the property when trying to avoid `event:/`